components/server/ee/src/container-module.ts

```diff
@@ -52,7 +53,11 @@ export const productionEEContainerModule = new ContainerModule((bind, unbind, is
     bind(PrebuildRateLimiter).toSelf().inSingletonScope();
     bind(PrebuildQueueMaintainer).toSelf().inSingletonScope();
     bind(IPrefixContextParser).to(StartPrebuildContextParser).inSingletonScope();
-    bind(GithubApp).toSelf().inSingletonScope();
+    bind(GithubApp).toDynamicValue(ctx => {
```

Comment:
Note: This might be irrelevant

Reply:
isn't it same as the ctor injection?
or did you try to solve an issue with eager dep resolution?